src/editortoolkit_cmn.cpp

```diff
@@ -247,15 +247,15 @@ bool EditorToolkitCMN::KeyDown(std::string &elementId, int key, bool shiftKey, b
 
     // For elements whose y-position corresponds to a certain pitch
     if (element->HasInterface(INTERFACE_PITCH)) {
-        PitchInterface *interface = element->GetPitchInterface();
-        assert(interface);
+        PitchInterface* pitch_interface = element->GetPitchInterface();
```

Comment:
I think you've just renamed this, without changing what it does?

Reply:
Yup.  The keyword 'interface' is the subject of a macro that expands it to "__STRUCT__", or at least my IDE was warning me of one.  So I renamed it to both be less vague, and avoid that overlap.

Reply:
You may need to configure your IDE to not do that, as the `*interface` name is used in a lot of places in Verovio.